src/test/cucumber/flush.feature

```diff
@@ -37,3 +37,12 @@ Feature: Flush
     When I enter "sdk flush temp"
     Then no "res-1.2.0.zip" file is present in temporary storage
     And I see "1 archive(s) flushed"
+
+  Scenario: Clear out the cached archives and the temporary space
```

Comment:
temporary storage?

Reply:
ok I'll implement those advice and push them :+1:

Reply:
by the way I was just orienting myself from the tests already written :-)

>  Scenario: Clear out the temporary space